include/cantera/transport/UnityLewisTransport.h

```diff
@@ -60,6 +60,13 @@ class UnityLewisTransport : public MixTransport
         }
     }
 
+    //! Note: thermo diffusion is not enabled in the unity Lewis number model
```

Comment:
```suggestion
    //! Thermal diffusion is not enabled in the unity Lewis number model.
```
You might also clarify that "not enabled" means that the returned coefficients are zero, to distinguish from other possibilities like raising `NotImplementedError`.